testdata/p4_16_samples_outputs/issue1914-1-first.p4

```diff
@@ -0,0 +1,67 @@
+#include <core.p4>
+
+struct standard_metadata_t {
+    bit<8> foo;
+}
+
+parser Parser<H, M>(packet_in b, out H parsedHdr, inout M meta, inout standard_metadata_t standard_metadata);
+control VerifyChecksum<H, M>(inout H hdr, inout M meta);
+control EmptyVerifyChecksum<H, M>(inout H hdr, inout M meta) {
+    apply {
+    }
+}
+
+control Ingress<H, M>(inout H hdr, inout M meta, inout standard_metadata_t standard_metadata);
+control Egress<H, M>(inout H hdr, inout M meta, inout standard_metadata_t standard_metadata);
+control ComputeChecksum<H, M>(inout H hdr, inout M meta);
+control Deparser<H>(packet_out b, in H hdr);
+package MySwitch<H, M>(Parser<H, M> p, VerifyChecksum<H, M> vr=EmptyVerifyChecksum<_, _>(), Ingress<H, M> ig, Egress<H, M> eg, ComputeChecksum<H, M> ck, Deparser<H> dep);
```

Comment:
This intermediate output for this test program has `EmptyVerifyChecksum<_, _>()`.  Is this expected?  Is there some brief explanation of why it is `<_, _>` instead of `<H, M>` for this test program, whereas most of the others have `<H, M>`, I think?

Reply:
Yes, this was the symptom of a bug.